pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -274,6 +271,47 @@ def _reraise_if_not_permitted(self, err: VerificationError, code: str, permits:
         self._logger.warn("Issue %s is permitted with justification: %s", err, justification)
         return justification
 
+    def validate_upgrades(self, upgrades_str: str, assembly_type: assembly.AssemblyTypes, build_data_path: os.PathLike) -> List:
```

Comment:
The name `validate_upgrades` does not immediately reveal what type of updates are being validated, the word is a bit overloaded. Maybe `validate_upgrade_edges`?

Reply:
:+1: